tests/test_datadog.py

```diff
@@ -42,3 +44,55 @@ def test_api_key():
     configured_api_key = DataDog.settings[DataDog.KEY_DATADOG_API_KEY]
 
     assert configured_api_key == 'test_api_key'
+
+
+def test_metrics():
+    metric_prefix = test_metrics.__name__
+    DataDog.configure_settings({'DATADOG_STATS_ENABLED': True,
+                                'DATADOG_STATS_PREFIX': metric_prefix})
+
+    mock_dd = DataDog.stats()
+    for method in ['gauge', 'increment', 'decrement', 'histogram', 'event']:
+        setattr(mock_dd, method, mock.Mock())
```

Comment:
Yes - I'm asserting on the methods of the mock, not the mock itself, and I wanted to make them separate instances of `Mock` so that I could use `assert_called_with`